testsuite/features/reposync/srv_enable_sync_products.feature

```diff
@@ -30,6 +30,19 @@ Feature: Be able to list available products and enable them
     And I should get "- sle-product-we15-sp3-pool-x86_64-sled"
     And I should get "Product successfully added"
 
+  Scenario: Enable "SUSE Linux Enterprise Desktop 15 SP4 x86_64" with recommended modules
```

Comment:
Should we remove previous scenario `Enable "SUSE Linux Enterprise Desktop 15 SP3 x86_64" with recommended modules`?

From what I see, you enable SP3 at two places: in the scenario before, and in`testsuite/features/reposync/srv_sync_products.feature`. Sounds like one too much.

Reply:
I do not know. Before we also had this code duplication with 15 SP3.
See
https://github.com/uyuni-project/uyuni/blob/4c43c57d0758142590a5ea4078c4ced42d02038e/testsuite/features/reposync/srv_enable_sync_products.feature#L20-L31

https://github.com/uyuni-project/uyuni/blob/4c43c57d0758142590a5ea4078c4ced42d02038e/testsuite/features/reposync/srv_sync_products.feature#L74-L93

Reply:
Can we agree on removing any kind of product synchronized that we will kill on next scenarios? I don't see any benefit, as we are already testing the feature to add products in other scenarios.
@Bischoff do you see any problem on completely remove scenarios synching not used products?

Reply:
@srbarrios :

I asked exactly same question long ago to Silvio Moioli and here is the answer I got at the time:

These scenarios are here to test connection to SCC and reposync itself. This is why we enable things completely unrelated to what we need, like SAP.

It might sound stupid to start reposyncs, and then kill them, but that's intentional.

Reply:
Maybe we should add a comment in our code to state this and to let everybody know why this is done.

Reply:
> I do not know. Before we also had this code duplication with 15 SP3. See

Hmmm, maybe we did not see the problem previously :-) . 

If we have to keep only one of both, I'd vote to keep the second, because it tests "recommended" functionality.

Reply:
> Maybe we should add a comment in our code to state this and to let everybody know why this is done.

Yes. Normally the scenario name should serve that purpose too, but yes an extra comment would not hurt...

I agree it is not clear:
- which products we currently enable to test some SCC / reposync / wizard feature (and the reposync will get killed)
- which products we currently enable because we will need them (and the reposync will NOT get killed)

Reply:
SLED is for sure nothing we need.
My proposal is to refactor all our scenarios about testing recommended, test scc connection, whatever else, so they use products that we really want to sync, and we get rid of any product sync triggered that we will then kill it later.

Reply:
Agreed but I would suggest doing this in another PR afterwards.

Reply:
I opened this card to address it https://github.com/SUSE/spacewalk/issues/18386 and link it as dependency for https://github.com/SUSE/spacewalk/issues/17645